crates/services/executor/src/executor.rs

```diff
@@ -210,8 +211,9 @@ impl ExecutionData {
 /// Per-block execution options
 #[derive(serde::Serialize, serde::Deserialize, Clone, Default, Debug)]
 pub struct ExecutionOptions {
+    // TODO: This is a bad name and the real motivation for this field should be specified
     /// UTXO Validation flag, when disabled the executor skips signature and UTXO existence checks
-    pub utxo_validation: bool,
+    pub extra_tx_checks: bool,
```

Comment:
This should simply be named after the situation we want to use it. I'm not sure what it is yet. For now, I just named it `extra_tx_checks` which is vague, but `utxo_validation` was incorrect, so it's an improvement.